docs/source/_src_user_guide/state_propagation/propagation_setup/propagation_architecture.rst

```diff
@@ -0,0 +1,81 @@
+.. _propagation_architecture:
+
+========================
+Propagation Architecture
+========================
+
+The core of a numerical integration is the succesive evaluation of the state derivative :math:`\dot{\mathbf{x}}=\mathbf{f}(\mathbf{x},s;\mathbf{p})`. Here, :math:`\mathbf{x}` is the propagated state which may be a combination of any types of dynamics of any set of bodies (see :ref:`TODO`), and may also include the variational equations associated with this dynamics. The variable :math:`s` is the indpendent variable of the differential equation governing the state (typically but not necesarilly the time :math:`t`), and the vector :math:`\mathbf{p}` denotes a set of parameters which are held constant during the propagation, but which do influence the solution of the equations of motion.
+
+.. _propagator_pre_processing:
+
+Propagator pre-processing
+=========================
+
```

Comment:
this subsection is an extremely useful piece of documentation, and the same goes for the post-processing part!